webpack/assets/javascripts/react_app/components/Layout/Layout.js

```diff
@@ -74,7 +75,7 @@ class Layout extends React.Component {
         onItemClick={primary =>
           handleMenuClick(primary, activeMenu, changeActiveMenu)
         }
-        activePath={`/${activeMenu}/`}
+        activePath={`/${__(activeMenu)}/`}
```

Comment:
I'm confused, why are we passing the translated string here? shouldn't the decision what item should be active depend on the path rather than the title?

Reply:
The VerticalNav's `onItemClick` method gives you the `primary, secondary, tertiary` menuItems that the clicked item belongs to, so it already gives you the correct title. Since we only care for the primary it can become much simpler with #6672

Reply:
so this won't be needed after GH-6672 is in? or is it unrelated?

Reply:
unrelated, we can merge this.